rl_studio/agents/settings.py

```diff
@@ -0,0 +1,41 @@
+###########################
+# Global variables file
+###########################
+from pydantic import BaseModel
+
+
+class QLearnConfig(BaseModel):
+    actions: int = 3
+    debug_level: int = 0
+    telemetry: bool = False
+    telemetry_mask: bool = True
+    plotter_graphic: bool = False
+    my_board: bool = True
+    save_positions: bool = False
+    save_model: bool = True
+    load_model: bool = False
```

Comment:
Maybe we could change the `BaseModel` by the `BaseSettings`. You can check this class in their documentation, [here](https://pydantic-docs.helpmanual.io/usage/settings/).